### DISCUSSION TITLE
Support `.config` dir

### DISCUSSION THREAD

**Discussion by kvnwolf** [biomejs/biome#1833] [Ideas]
There has been recently community complaints about the amount of configuration files present on the root directory of a project, and some people are recommending to place all tooling configuration under a `.config` directory.

I personally think that, if Biome starts adding support for a `.config` dir, more projects could start adopting the practice too, since Biome is growing in popularity and encouraging reducing configuration overhead.

Reference: https://github.com/pi0/config-dir

**Comment by mkvlrn** [comment]
Harmless "2 years after initial suggestion" bump.

**Reply by ematipico** [reply]
https://biomejs.dev/guides/configure-biome/#configuration-file-resolution

**Reply by mkvlrn** [reply]
I'm sorry, I'm not seeing it: in that guide, it says it will obey `$XDG_CONFIG_HOME`, but that is not the same as the proposal from https://github.com/pi0/config-dir.

The first is in the user's home directory, outside of the project. The second is in the root of the project, but inside a `.config` directory.

**Reply by ematipico** [reply]
Ah `.config/` inside the project.
